lib/msal-node/test/client/ManagedIdentitySources/Imds.spec.ts

```diff
@@ -549,13 +550,136 @@ describe("Acquires a token successfully via an IMDS Managed Identity", () => {
             );
         });
 
-        test("ignores a cached token when claims are provided", async () => {
+        test('checks if a token was cached or not and if was "bypass_cache=true" was sent to ESTS depending on whether or not claims are provided, and ensures "xms=client-capabilites-string" was sent to ESTS when client capabilities are provided via the Managed Identity Request Parameters', async () => {
+            const sendGetRequestAsyncSpy: jest.SpyInstance = jest.spyOn(
+                networkClient,
+                <any>"sendGetRequestAsync"
+            );
+
             let networkManagedIdentityResult: AuthenticationResult =
                 await systemAssignedManagedIdentityApplication.acquireToken({
+                    clientCapabilities: CAE_CONSTANTS.CLIENT_CAPABILITIES,
```

Comment:
ClientCapabilities should be set only at the "app" / "client" level, not at the request level.